salt/modules/yaml.py

```diff
@@ -0,0 +1,50 @@
+"""
+Yaml helper module for troubleshooting yaml
+
+"""
+
+import logging
+
+log = logging.getLogger(__name__)
+
+__virtualname__ = "yaml"
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def lint(source, saltenv=None, pre_render=None, **kwargs):
+    """
+    lint the output after detecting a sucsessful render.
+
+    :param str source: managed source file
+
+    :param str saltenv: the saltenv to use, defaults
+        to minions enviroment or base if not set
+
+    :param str pre_render: The render options passed to
+        slsutil.renderer other wise file is cached and loaded as stream
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' yamllint.lint salt://example/bad_yaml.sls
+    """
+    if saltenv is None:
+        saltenv = __salt__["config.get"]("saltenv", "base")
+        if saltenv is None:
```

Comment:
since you are setting the default to be "base" in the config.get call is there any case where saltenv would be none here still?